crates/store/src/backend/foundationdb/write.rs

```diff
@@ -109,6 +109,7 @@ impl FdbStore {
                                 }
                             }
                             ValueOp::AtomicAdd(by) => {
+                                println!("fdb atomic add key: {:?} val: {:?} ", key, by.to_le_bytes());
```

Comment:
You forgot a println here.

Reply:
Oops. I was (and still am) figuring out how to do atomic adds with TiKV.